web/grunt/karma-config.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function (grunt) {
+    'use strict';
+
+    const unitTests =  grunt.config('config.tests');
+
+    const karmaTasksConfig = {
+        browsers: ['Firefox'],
+        frameworks: ['jasmine', 'browserify'],
+        preprocessors: {
+            [unitTests]: ['browserify'],
```

Comment:
Use object-inline grunt config processing instead of this, so this can match the line below. That means you can remove `unitTests = grunt.config('config.tests').